test/TestCase.cpp

```diff
@@ -116,19 +116,35 @@ bool EVMVersionRestrictedTestCase::validateSettings(langutil::EVMVersion _evmVer
 	if (versionString.empty())
 		return true;
 
-	char comparator = versionString.front();
-	versionString = versionString.substr(1);
+	string comparator;
+	size_t versionBegin = 0;
+	for (auto character: versionString)
```

Comment:
Since you have the index counter anyway, perhaps a for loop with `isalpha(versionString[i])` would be more intuitive? But it can also stay as it is.